brands.md

```diff
@@ -4,7 +4,7 @@ After creating an account and getting access to the dashboard, the next step is
 
 The **Brand** object is used to aggregate locations and keep track of the Brand consent. To track real-time transactions at a Brand's locations, you need to obtain consent from authorised personnel (Brand Contact), and provide the Brand with access to view transactional data. You can do this when creating the Brand (see below).
 
-Once you create a **Brand**, it cannot be deleted.
+Once you create a **Brand**, it cannot be deleted in the Dashboard. In live mode you can use the [Delete Brand](https://reference.fidel.uk/reference/delete-brand) endpoint to delete brands that have no signed consent yet.
```

Comment:
Should we say something after "in the Dashboard." Like "Using the API, in live mode, you can use..."? It's because the dashboard has concepts like live and test mode as well and I wonder if they might get conflated by users (I had to read it twice myself)

Reply:
Note that this is not true, if you remove the consent on the dashboard you also can delete the brand in live mode.

Reply:
I would argue that this is dangerous to allow clients to do it because could cause other issues like what would happen to locations and offers from the deleted brand, which is probably why the docs say it cannot be deleted, we probably should even remove the permission to do it until we have a clean up job that removes the resources that depend on the brand.

Reply:
also to clarify  this 
> (...) dashboard has concepts like live and test mode as well (...)

the same concept also exists on the api and is used with the keys you choose, if you use the `pk_test_*` or `sk_test_*` keys it's test mode if you use `pk_live_*` or `sk_live_*` it's live mode

Reply:
Ok, removing my changes about the deletion, according to what Diogo suggested. If the behavior is not recommended, please consider making that endpoint unavailable to public.